.github/workflows/deploy-to-aws.yml

```diff
@@ -1,18 +1,19 @@
 name: Deploy to AWS
 
 on:
-  workflow_dispatch:
-    inputs:
-      environment:
-        description: 'The environment to deploy to'
-        required: true
-        default: 'dev'
-  #push:
-  #  branches: ["dev"]
+#  workflow_dispatch:
+#    inputs:
+#      environment:
+#        description: 'The environment to deploy to'
+#        required: true
+#        default: 'dev'
```

Comment:
I would remove this rather than leaving it commented out if it is not longer useful.

Reply:
Yes, makes sense. I'll wait until George approves this approach or not, before deleting it. Perhaps he'd like to keep the manual approach as well. Then I uncomment it.